### DISCUSSION TITLE
phase_a, phase_b, phase_c nomenclature

### DISCUSSION THREAD

**Discussion by lumbarhopkins** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#32] [Q&A]
I may be overthinking this, but shouldn't phase_a, phase_b and phase_c nomenclature really be channel_a, channel_b and channel_c for each ATM90E32 IC as each input is on the same phase (CT1, CT2 and CT3 on supply voltage and CT4, CT5 and CT6 on VA2 assuming JP12/JP13 severed)?

**Comment by CircuitSetup** [comment]
Ideally they would be called channels, yes. Originally, the IC we use is made for 3 phase measurements, so in the datasheets they refer to the channels as phases.
